powerhome/playbook
# [PLAY-641] ♻️ Typescript Conversion: Multiple Users Kit

#### Screens

<img width="190" alt="Screen Shot 2023-03-06 at 10 59 42 AM" src="https://user-images.githubusercontent.com/73671109/223163655-0ee95396-8d00-4ff2-84db-3b4dff5f44d9.png">

#### Breaking Changes

NO

#### Runway Ticket URL

[Runway Ticket](https://nitro.powerhrg.com/runway/backlog_items/PLAY-641)

#### How to test this

Use multiple users kit and ensure it works as it should after typescript conversion

#### Checklist:
